client/components/table/style.scss

```
@@ -50,6 +51,16 @@
 	}
 }
 
+.woocommerce-table__shadow {
+	content: '';
+	position: absolute;
+	right: 0;
+	top: 0;
+	width: 41px;
+	height: 100%;
+	background: linear-gradient(90deg, rgba(0, 0, 0, 0), rgba(0, 0, 0, 0.1), rgba(0, 0, 0, 0.2));
```

The color in the middle shouldn't be necessary:
```suggestion
	background: linear-gradient(90deg, rgba(0, 0, 0, 0), rgba(0, 0, 0, 0.2));
```

---

Also correct- removed!